modules/apps/object/object-api/src/main/java/com/liferay/object/util/ObjectActionDataConverterUtil.java

```diff
@@ -15,17 +15,58 @@
 package com.liferay.object.util;
 
 import com.liferay.object.model.ObjectDefinition;
+import com.liferay.portal.kernel.json.JSONFactoryUtil;
 import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.util.GetterUtil;
+import com.liferay.portal.kernel.util.UnicodeProperties;
+import com.liferay.portal.kernel.util.UnicodePropertiesBuilder;
 import com.liferay.portal.vulcan.dto.converter.DTOConverter;
 import com.liferay.portal.vulcan.dto.converter.DTOConverterRegistry;
 
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.Map;
+import java.util.Objects;
 
 /**
  * @author Carolina Barbosa
  */
 public class ObjectActionDataConverterUtil {
 
+	public static UnicodeProperties convertParameters(
+		Map<String, ?> parameters) {
+
+		return UnicodePropertiesBuilder.create(
+			_toStringMap(parameters), true
+		).build();
+	}
+
+	public static Map<String, Object> convertParametersUnicodeProperties(
```

Comment:
Do you think it makes sense since everywhere else, in the Objects code, it's called `ParametersUnicodeProperties`? As in `ObjectActionLocalServiceImpl`.